sip-core/src/main/java/de/ikor/sip/foundation/core/trace/SIPTraceConfig.java

```diff
@@ -0,0 +1,20 @@
+package de.ikor.sip.foundation.core.trace;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/** Configuration class to read from property file */
+@Getter
+@Setter
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "sip.core.tracing")
+public class SIPTraceConfig {
+
+  private int limit;
+  private String traceType;
+  private boolean enabled;
```

Comment:
Since we introduced traceType, enabled seams to be redundant. 
Let's introduce another traceType value for none instead

Reply:
ok